lib/ruote/mon/storage.rb

```diff
@@ -53,11 +53,14 @@ def initialize(mongo_db, options={})
         # can't do that...
 
       (TYPES - %w[ msgs schedules ]).each do |t|
-        collection(t).ensure_index('_wfid')
-        collection(t).ensure_index([ [ '_id', 1 ], [ '_rev', 1 ] ])
+        collection(t).indexes.create_one('_wfid' => 1)
+        collection(t).indexes.create_many([
+          { key: {'_id' => 1}},
+          { key: {'_rev' => 1}}
```

Comment:
I'll check tomorrow, but I think this is supposed to be one index over two columns.